l10n_es_aeat_vat_prorrate/views/mod303_view.xml

```diff
@@ -17,12 +17,12 @@
                 <field name="type" position="after">
                     <field name="vat_prorrate_type"/>
                     <label for="vat_prorrate_percent"
-                           attrs="{'invisible': [('vat_prorrate_type', '!=', 'general')]}"
+                           attrs="{'invisible': [('vat_prorrate_type', 'not in', ['general', 'special'])]}"
                             />
                     <div class="oe_inline"
-                         attrs="{'invisible': [('vat_prorrate_type', '!=', 'general')]}">
+                         attrs="{'invisible': [('vat_prorrate_type', 'not in', ['general', 'special'])]}">
                         <field name="vat_prorrate_percent"
-                               attrs="{'required': [('vat_prorrate_type', '=', 'general')]}"
+                               attrs="{'required': [('vat_prorrate_type', 'not in', ['general', 'special'])]}"
```

Comment:
Rectifico, sí que había que añadir el special, por tanto sí que había que modificarlo pero usarndo el 'in'